beets/dbcore/db.py

```diff
@@ -1057,6 +1130,20 @@ def _make_attribute_table(self, flex_table):
                     ON {0} (entity_id);
                 """.format(flex_table))
 
+    def _make_list_tables(self, list_tables, list_types):
+        """Create tables for list types.
+        """
+        with self.transaction() as tx:
+            for key, name in list_tables.items():
+                tx.script(f"""
```

Comment:
Isn't this supposed to be unsafe? i.e. one should use `.format` instead of f-strings? It would in any case be more consistent as this is what you do in the other sql strings. But this is just a guess as I'm only reading trough this PR because it seems like a really nice function to have and I don't have a lot of knowledge of the beets codebase.